src/scenarios/scenarios_2.ts

```diff
@@ -3,12 +3,47 @@ import {ScenarioInterface} from "./scenario-interface";
 
 export class Scenarios_2 implements ScenarioInterface {
   async run() {
-    /*
+        /*
```

Comment:
Let's add a short scenarios description in README and remove the comment from here.

Reply:
Added to readme